examples/segnet/evaluate.sh

```diff
@@ -0,0 +1,2 @@
+python calc_bn_statistics.py --gpu $1 --snapshot $2
+python evaluate.py --gpu $1 --snapshot $2_infer
```

Comment:
How about calling `calc_bn_statistics` from `evaluate.py`? I think `*.sh` is not good for windows users.